src/main/java/controller/RaceController.java

```diff
@@ -1,30 +1,60 @@
 package controller;
 
 
+import model.Cars;
 import model.Race;
+import model.Round;
+import view.InputView;
 import view.OutputView;
 
 public class RaceController {
 
-    private final InputController inputController;
+    private final InputView inputView;
+    private final OutputView outputView;
 
-    public RaceController() {
-        this.inputController = new InputController();
+    public RaceController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
     }
 
     public void startRace() {
-        Race race = inputController.makeRace();
-        int tryCount = inputController.makeTryCount();
+        Race race = makeRace();
+        Round round = makeTryCount();
 
-        OutputView.showRaceResult();
-        runOneRound(tryCount, race);
-        OutputView.showWinners(race.selectWinners());
+        outputView.showRaceResult();
+        startRound(round, race);
+        outputView.showWinners(race.selectWinners());
     }
 
-    private void runOneRound(int tryCount, Race race) {
-        for (int round = 0; round < tryCount; round++) {
-            race.oneGame();
-            OutputView.showRoundResult(race.getCarsName(), race.getDistances());
+    private Race makeRace() {
+        try {
+            String carNames = inputView.inputCarName();
+            Cars cars = new Cars(carNames);
+
+            return new Race(cars);
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+
+            return makeRace();
+        }
+    }
+
+    private Round makeTryCount() {
+        try {
+            String inputRound = inputView.inputRound();
+
+            return new Round(inputRound);
```

Comment:
```suggestion
            String inputRound = inputView.inputRound();
            return new Round(inputRound);
```

혹시 return 하기 전에 개행을 해주신 이유가 있을까요?

Reply:
줄 바꿈도 컨벤션이다 라는 글을 보고 앞서 처리하던 로직과 반환 값을 분리하고 싶어 개행을 했습니다..!